src/pages/guides/using/creating_rest_apis.md

```diff
@@ -153,6 +153,145 @@ curl -i -X OPTIONS https://adobeioruntime.net/...
 
 ## Securing the API endpoints
 
+### Oauth (using the Adobe Identity Management System)
+
+You can configure an action to require IMS validation for incoming requests using the following command: 
```

Comment:
I'm thinking, maybe this step should go after the swagger configuration? This would be more natural in case somebody is actually going to migrate an existing action in production. If they do this step first, the action will be inaccessible until next step is done. So anyways whoever is reading this doc, would have to perform the steps backwards to avoid their service interruption.

Reply:
I've updated the command in the documentation from `wsk update` to `wks create`,  so that customers are not tempted to update existing actions.